### PR TITLE
Create <Tag> Component

### LINKED ISSUE #28: Create <Tags /> Component
[Design System Link](https://www.notion.so/h4iuiuc/Hack4Impact-s-Design-System-Bridge-40aaae97c71f400084b4bbcc5cd8cdee#473b7948612c47a886c22b4d848c71cc)

I'm not exactly sure how to name these color variants since we have success, error, warning stuff with buttons. Maybe just the name of the color itself? green, blue, red... and then have another prop for "light", "primary", "dark"

image

### PR DESCRIPTION
**Pull request by davidenwang**
Creates a <Tag> Component

has props for 
* 'color' (green, blue, red, etc...)
* 'variant' (light, primary, dark)

Resolves #28

### CODE REVIEW

File: src/components/Tag/Tag.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+const TagBase = styled.div`
+    // text
+    ${get('typography.body')};
+    color: ${(props) => props.textColor};
+
+    // border
+    border-radius: 4px;
+
+    // display
+    display: inline-flex;
```
Comment (tko22) [inline src/components/Tag/Tag.js:17]: nice

File: src/components/Tag/Tag.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  // both light green and light yellow tags have dark palette text
+  if (variant === 'light' && (color === 'green' || color === 'yellow')) {
+    textColor = lodashGet(propTheme, `colors.${color}Palette.text`, propTheme.colors.black);
+  }
+
+  return (
+    <TagBase
+      backgroundColor={backgroundColor}
+      textColor={textColor}
```
Comment (tko22) [inline src/components/Tag/Tag.js:49]: can you have `theme={propTheme}` to TagBase - this allows `get` to get from the theme passed in, or fallback to the default theme

**Comment by tko22** [comment]
WAIT WOOPS i added one tiny comment thats it

File: src/components/Tag/Tag.js (hunk 2 above)
Comment (davidenwang) [reply src/components/Tag/Tag.js:49]: oh true good catch, i'll update

File: src/components/Tag/Tag.js (hunk 1 above)
Comment (davidenwang) [reply src/components/Tag/Tag.js:17]: lol just learned about this for centering text vertically

**Comment by tko22** [comment]
good to merge whenever you want!

**Pull request merged**